doc/source/releases/15.10_announce.rst

```diff
@@ -0,0 +1,9 @@
+===========================================================
+October 2015 Galaxy Release (v 15.10)
```

Comment:
the planned release was only in `dev` last time, if we put it in the `release_15.07' the latest actual release will never be on top